MeshRender/OpenGLTest/Renderer.cpp

```diff
@@ -0,0 +1,135 @@
+#include "Renderer.h"
+extern GLFWwindow* window;
+
+
+Renderer::Renderer()
+{
+
+}
+
+
+
+Renderer::~Renderer()
+{
+}
+
+int Renderer::loop(CommonMeshRepresentation common_mesh)
+{
+//	VertexLoader pointArray("temp.xyz");
+//	MeshIS::Model::Common::CommonMeshRepresentation common_mesh;
+	int verticesArraySize = common_mesh.vertices.size()*3;
+	const float* arr = commonToArray(common_mesh);
```

Comment:
Pointer is not an array. http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#r11-avoid-calling-new-and-delete-explicitly